### PR TITLE
[Keyboard] Add epssp75

### PR DESCRIPTION
**Pull request by hen-des**
## Description

Initial upload of the files for the epssp75 keyboard

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [ ] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [ ] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/handwired/epssp75/info.json
Comment (fauxpark) [inline keyboards/handwired/epssp75/info.json:1]: This file should be renamed to `keyboard.json`. Please also run `qmk format-json -i`.

File: keyboards/handwired/epssp75/rules.mk
Comment (fauxpark) [inline keyboards/handwired/epssp75/rules.mk:1]: This file can be deleted.

File: keyboards/handwired/epssp75/keymaps/noted/keymap.c
Comment (fauxpark) [inline keyboards/handwired/epssp75/keymaps/noted/keymap.c:1]: User keymaps are no longer accepted, please remove this.

**Review by fauxpark — CHANGES_REQUESTED** [review]

File: keyboards/handwired/epssp75/info.json [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+		"rows": ["GP1", "GP2", "GP3", "GP4", "GP5"]
+	},
+	"processor": "RP2040",
+	"usb": {
+		"device_version": "1.0.0",
+		"pid": "0x0075",
+		"vid": "0x6863"
+	},
+	"layouts": {
+		"LAYOUT_epssp_75": {
```
Comment (waffle87) [inline keyboards/handwired/epssp75/info.json:27]:
```suggestion
		"LAYOUT": {

```

File: keyboards/handwired/epssp75/keymaps/default/keymap.c [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+		KC_CAPS,	KC_A,		KC_S,		KC_D,		KC_F,		KC_G,		KC_DEL,		KC_HOME,	KC_UP,		KC_END,		KC_BSPC,	KC_H,		KC_J,		KC_K,		KC_L,		KC_SCLN,	KC_QUOT,
+		KC_LSFT,	KC_Z,		KC_X,		KC_C,		KC_V,		KC_B,					KC_LEFT,	KC_DOWN,	KC_RGHT,				KC_N,		KC_M,		KC_COMM,	KC_DOT,		KC_SLSH,	KC_RSFT,
+		KC_LCTL,	KC_LGUI,	KC_LALT,	KC_NUBS,			KC_SPC,			KC_LSFT,										KC_ENT,			KC_SPC,				KC_RALT,	MO(1),		RGUI_T(KC_APP),KC_RCTL
+    ),
+	[1] = LAYOUT_epssp_75(
+		KC_ESC,		KC_F1,		KC_F2,		KC_F3,		KC_F4,		KC_F5,		KC_F22,			KC_PAUS,	KC_PSCR,			KC_F23,		KC_F6,		KC_F7,		KC_F8,		KC_F9,		KC_F10,		KC_F11,
+		KC_TAB,		KC_PGUP,	LCTL(KC_LEFT),KC_UP,	LCTL(KC_RGHT),KC_PGDN,	KC_F20,					KC_ENT,					KC_F21,		KC_NO,		KC_NO,		KC_NO,		KC_NO,		KC_NO,		KC_F12,
+		KC_CAPS,	KC_HOME,	KC_LEFT,	KC_DOWN,	KC_RGHT,	KC_END,		KC_DEL,		KC_HOME,	KC_VOLU,	KC_END,		KC_BSPC,	KC_NO,		KC_NO,		KC_NO,		KC_NO,		KC_NO,		KC_QUOT,
+		KC_LSFT,	LCA(KC_F2),	KC_TAB,		KC_F2,		KC_F5,		KC_F23,					KC_MUTE,	KC_VOLD,	KC_MUTE,				KC_NO,		KC_NO,		KC_NO,		KC_NO,		KC_NO,		KC_RSFT,
+		KC_LCTL,	KC_LGUI,	KC_LALT,	KC_NUBS,			KC_SPC,			KC_LSFT,										KC_ENT,			KC_SPC,				KC_RALT,	KC_NO,		KC_APP,		KC_RCTL
```
Comment (waffle87) [inline keyboards/handwired/epssp75/keymaps/default/keymap.c:19]:
```suggestion
		KC_LCTL,	KC_LGUI,	KC_LALT,	KC_NUBS,			KC_SPC,			KC_LSFT,										KC_ENT,			KC_SPC,				KC_RALT,	KC_NO,		_______,		KC_RCTL
```

File: keyboards/handwired/epssp75/keymaps/default/keymap.c [hunk 3]
```diff
@@ -0,0 +1,21 @@
+// Copyright 2023 QMK
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#include QMK_KEYBOARD_H
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+    [0] = LAYOUT_epssp_75(
```
Comment (waffle87) [inline keyboards/handwired/epssp75/keymaps/default/keymap.c:7]:
```suggestion
    [0] = LAYOUT(
```

File: keyboards/handwired/epssp75/keymaps/default/keymap.c [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+		KC_ESC,		KC_1,		KC_2,		KC_3,		KC_4,		KC_5,		KC_F22,			KC_RBRC,	KC_PSCR,			KC_F23,		KC_6,		KC_7,		KC_8,		KC_9,		KC_0,		KC_MINS,
+		KC_TAB,		KC_Q,		KC_W,		KC_E,		KC_R,		KC_T,		KC_F20,					KC_ENT,					KC_F21,		KC_Y,		KC_U,		KC_I,		KC_O,		KC_P,		KC_LBRC,
+		KC_CAPS,	KC_A,		KC_S,		KC_D,		KC_F,		KC_G,		KC_DEL,		KC_HOME,	KC_UP,		KC_END,		KC_BSPC,	KC_H,		KC_J,		KC_K,		KC_L,		KC_SCLN,	KC_QUOT,
+		KC_LSFT,	KC_Z,		KC_X,		KC_C,		KC_V,		KC_B,					KC_LEFT,	KC_DOWN,	KC_RGHT,				KC_N,		KC_M,		KC_COMM,	KC_DOT,		KC_SLSH,	KC_RSFT,
+		KC_LCTL,	KC_LGUI,	KC_LALT,	KC_NUBS,			KC_SPC,			KC_LSFT,										KC_ENT,			KC_SPC,				KC_RALT,	MO(1),		RGUI_T(KC_APP),KC_RCTL
+    ),
+	[1] = LAYOUT_epssp_75(
```
Comment (waffle87) [inline keyboards/handwired/epssp75/keymaps/default/keymap.c:14]:
```suggestion
	[1] = LAYOUT(
```

File: keyboards/handwired/epssp75/readme.md [hunk 5]
```diff
@@ -0,0 +1,13 @@
+# epssp75
+
+This is an Ergonomic PSeudo SPlit 75-key Keyboard. It was desinged after two years of working with a [Truly Ergonomic CLEAVE Keyboard](https://trulyergonomic.com) and is also inspired by the [Redox Keyboard](https://github.com/mattdibi/redox-keyboard).
+The layout I am using and shown in the image is the [NOTED](https://github.com/dariogoetz/noted-layout) layout.
+
+In addition to the firmware created by QMK, I am using the [ReNeo](https://neo-layout.org/Einrichtung/reneo/) software which offers more possible key functions and in total six layers.
+
+![epssp75](https://raw.githubusercontent.com/hen-des/epssp75/main/docs/assets/epssp75-Noted.jpg)
```
Comment (waffle87) [inline keyboards/handwired/epssp75/readme.md:8]: Please rehost this image on Imgur.

**Review by waffle87 — CHANGES_REQUESTED** [review]
In the future, please use the provided pull request template and fill it out.

**Pull request closed without merging**